app/workers/asset_manager_create_whitehall_asset_worker.rb

```diff
@@ -1,8 +1,8 @@
 class AssetManagerCreateWhitehallAssetWorker < WorkerBase
   def perform(file_path, legacy_url_path)
     file = File.open(file_path)
-    asset_options = {file: file, legacy_url_path: legacy_url_path}
-    asset_options[:draft] = true if legacy_url_path =~ /attachment_data/
+    asset_options = { file: file, legacy_url_path: legacy_url_path }
+    asset_options[:draft] = true if legacy_url_path.match?(/attachment_data/)
```

Comment:
Minor: These changes seem orthogonal to the commit note. I think they'd be better combined into the first commit.

Reply:
Good spot. I incorrectly added it to this commit. I've now moved it to the previous commit.